_posts/2023-03-13-llvm-opaque-pointers.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "LLVM opaque pointer support has landed"
+author: HertzDevil
+summary: 
+---
+
+Crystal 1.8, the upcoming minor release, will support LLVM's opaque pointers for the first time, allowing the compiler to be built with LLVM 15 or above. To understand the significance of opaque pointers, let's take a look at a small sample program:
```

Comment:
The speedup is the most important message for most Crystal users, let's mention that right at the start.

```suggestion
Crystal 1.8, the upcoming minor release, will support LLVM's opaque pointers for the first time, allowing the compiler to be built with LLVM 15 or above. Additionally, this update brings a significant improvement to compilation times.

## Pointers in LLVM

To understand the significance of opaque pointers, let's take a look at a small sample program:
```